auto_submit/lib/service/validation_service.dart

```diff
@@ -368,6 +372,41 @@ Exception: ${exception.message}
     return ProcessMergeResult.noMessage(true);
   }
 
+  /// A user can provide an alternate merge method via comment by providing the
+  /// string @autosubmit:<merge_method>.
+  static final RegExp regExpMergeMethod = RegExp(r'@autosubmit:(merge|squash|rebase)', caseSensitive: false);
+  Future<github.MergeMethod> checkForMergeMethod({
+    required github.RepositorySlug slug,
+    required int prNumber,
+  }) async {
+    const github.MergeMethod defaultMergeMethod = github.MergeMethod.squash;
+    final GithubService githubService = await config.createGithubService(slug);
+
+    final List<github.IssueComment> issueComments = await githubService.listIssueComments(slug, prNumber);
+    if (issueComments.isEmpty) {
+      return defaultMergeMethod;
+    }
+
+    final List<github.MergeMethod> foundMatches = [];
+    for (github.IssueComment comment in issueComments) {
```

Comment:
Are issueComments going to be sorted as expected everytime?

Reply:
Yes, issue comments are sorted by ascending id.
https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments